content/guides/references/migration-guide.md

```diff
@@ -2,7 +2,361 @@
+<Badge type="success">After</Badge> One big test using `cy.origin()`
+
+```js
+it('securely edits content', () => {
+  cy.origin('supersecurelogons.com', () => {
+    cy.visit('https://supersecurelogons.com')
+    cy.get('input#password').type('Password123!')
+    cy.get('button#submit').click()
+  })
+
+  cy.origin('mycms.com', () => {
+    cy.url().should('contain', 'cms')
+    cy.get('#current-user').contains('logged in')
+    cy.get('button#edit-1').click()
+    cy.get('input#title').type('Updated title')
+    cy.get('button#submit').click()
+    cy.get('.toast').type('Changes saved!')
+  })
+
+  cy.visit('/items/1')
+  cy.get('h1').contains('Updated title')
+})
+```
+
+The just-release `cy.session()` command can be used to setup and cache cookies,
+local storage and session storage between tests to easily re-establish the
+previous (or common) browser contexts needed in a suite. This command will run
+setup on the initial execution and will restore the saved browser state on each
+sequential command execution. This command reduces the need for repeated
+application logins, while users also benefit from the test isolation guardrails
+to write independent, reliable and deterministic tests from the start.
+
+If for whatever reason you still need to persist the dom and browser context
+between tests, you can set `testIsolation='off` on the root configuration or at
+the suite-level. For example:
+
+```js
+describe('workflow', { testIsolation: 'off' }, () => {
+  ...
+})
+```
+
+It is important to note that while turning test isolation `off` may improve the
+overall performance of end-to-end tests, previous tests could be impact the
+browser state. It is important to be extremely mindful of how test are written
+when using this mode and ensure tests continue to run independent from one
+other.
+
+<Badge type="danger">For example</Badge>the following tests are not independent
+nor deterministic:
+
+```js
+describe('workflow', { testIsolation: 'off' }, () => {
+  it('logs in', () => {
+    cy.visit('my-app.com/log-in)
+    cy.get('username').type('User1')
+    cy.get('password').type(Cypress.env('User1_password'))
+    cy.get('button#login').click()
+    cy.contains('User1')
+  })
+
+  it('clicks user profile', () => {
+    cy.get('User1').find('#profile_avatar).click()
+    cy.contains('Email Preferences')
+  })
+
+  it('updates profile', () => {
+    cy.get('button#edit')
+    cy.get('email').type('user1@email.com')
+    cy.get('button#save').click()
+  })
+})
+```
+
+In the above example, each test is relying on the previous test to be
+_successful_ to correctly execute. If at any point, the first or second test
+fails, the sequential test(s) will automatically fail and provided un-reliable
+debugging errors since the errors are representative of the previous test.
+
+The best way to ensure your tests are independent is to add a `.only()` to your
+test and verify it can run successfully without the test before it.
+
+### Alias Behaviors Changes
```

Comment:
To me this is a slightly confusing heading. Maybe change to "Changes to behavior of aliases"?